testcases/OpTestMtdPnorDriver.py

```diff
@@ -107,7 +107,7 @@ def runTest(self):
 
         # Getting the /tmp/pnor file into local x86 machine
         l_path = "/tmp/"
-        self.util.copyFilesToDest(l_path, self.host_user, self.host_ip, l_file, self.host_Passwd, "2", BMC_CONST.SCP_TO_LOCAL)
+        self.util.copyFilesToDest(l_path, self.host_user, self.host_ip, l_file, self.host_Passwd)
         l_list =  commands.getstatusoutput("ls -l %s" % l_path)
```

Comment:
But this one is actually to copy the PNOR from SUT to local machine. For which we need this PR #246

Reply:
I just fixed number of args to that function. Its fine to wait until PR @246 gets merged.

-Vasant